generic_config_updater/change_applier.py

```diff
@@ -8,8 +8,8 @@
 from swsscommon.swsscommon import ConfigDBConnector
 from .gu_common import genericUpdaterLogging
 
-
-UPDATER_CONF_FILE = "/etc/sonic/generic_config_updater.conf"
```

Comment:
>generic_config_updater.conf [](http://example.com/codeflow?start=32&length=27)

Regarding the filename, `generic_config_updater.conf.json` seems make more sense. It is confusing to see `..updater.conf..`